### PR TITLE
feat: add GHCR container images

### PR DESCRIPTION
**Pull request by onedr0p**
Fixes: #381

### CODE REVIEW

**Comment by onedr0p** [comment]
@zoriya after this is reviewed and merged you might have to make the packages public by going to https://github.com/zoriya?tab=packages

File: .github/workflows/docker.yml
Comment (onedr0p) [inline .github/workflows/docker.yml:99]: No need to create a secret, this is auto generated.

**Review by zoriya — APPROVED** [review]

**Comment by zoriya** [comment]
Do I need to create a package beforehand, or will the action create it for me?

**Comment by Roxedus** [comment]
It will create it

**Comment by Roxedus** [comment]
🙏 https://github.com/zoriya/Kyoo/pkgs/container/kyoo_back

**Comment by zoriya** [comment]
the "latest" tag will only be available after the next tagged release, but the edge tag will follow master!

**Comment by onedr0p** [comment]
As a follow-up (not precisely any time soon) would you be okay to have this be the default in the docker compose files?

**Comment by zoriya** [comment]
Yes, especially if it's better on all points. Will need to wait till next tagged release but after that I'm okay with that.

**Pull request merged**
